src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryActivator.java

```diff
@@ -123,7 +123,8 @@
     private volatile ResourceResolverFactoryConfig config = DEFAULT_CONFIG;
 
     /** Alias path whitelist */
```

Comment:
Comment still contains the term "whitelist". I would argue the comment is not needed, because the field's name describes its purpose sufficiently.